src/NServiceBus.AcceptanceTests/DelayedDelivery/When_deferring_a_message_to_the_past.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NServiceBus.AcceptanceTests.DelayedDelivery
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+
+    public class When_deferring_a_message_to_the_past : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_deliver_message_now()
```

Comment:
I think that it is not clear what is the purpose of this test. My understanding is that primary goal is to prove that exception is not thrown. If I'm right can we change the name to reflect that? Also this test proves that message will be delivered in test timeout period which is 20 sec which makes 'now' a bit misleading as well.